ILIASObject: tile image export path fix

chlulei opened the pull request:
LINK: https://mantis.ilias.de/view.php?id=45574

This PR fixes the tile-image export path.

chlulei committed:
ILIASObject: tile image export path fix

kergomard commented:
Thank you very much for the PR @chlulei !

I tested this and it seems to not be enough. It works now for the top level, but on nested object it still crashes upon import. E.g. create a Category inside a category and assign a tile image to each. This will export file, but will crash upon import with the trace below. On more complex structures, with different object types, I got it to crash on exports, but this might have been due to other issues, so I stick to this.

Best,
@kergomard

chlulei committed:
ILIASObject: fix tile image export path during container export

chlulei commented:
@kergomard,

thank you again for analyzing this PR. I updated the PR to fix the error that occurs during container exports. The tile image file is now placed in the correct folder during container exports. To achieve this i added a condition to check if the current export is a container export and adjusted the export path accordingly.

Best,
@chlulei

kergomard commented:
Thank you very much for the PR @chlulei ! This seems to do the trick. Merged and picked to ILIAS 11 and trunk.
